src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -91,6 +91,7 @@ const en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonContent) => ({
   },
 });
 
+// @TODO translations
```

Comment:
```suggestion
```

Reply:
Thanks @lara-mpt I removed that translation comment.